### DISCUSSION TITLE
SUGERENCIA

### DISCUSSION THREAD

**Discussion by JoseBenin82** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#282] [Ideas]
A decir verdad en lo personal me hubiera gustado que  nos dieran al inicio las intrucciones de descarga de las extensiones de HTML, realmente me huviera gustadocomo en el curso de introduccion de Python. Solo es mi opinion personal.

**Comment by JuanSalvador73** [comment]
Bueno, aunque es una interesante sugerencia algo que el MC ha mencionado seguido- y la instructora anterior también - es que en esta capacitación laboral uno debe de poner mucha iniciativa de nuestra parte. Personalmente creo que por eso mismo muchos de los temas explicados parecen que les falta algo.

Lo que nos están dando aquí es una "probada" de como trabaja el FrontEnd en Web para que uno estudie por su cuenta mas a fondo como trabaja esta tecnología. En mis años de trabajo por lo general es así. A uno lo contratan para solucionar problemas.

Y a mi ver va bien. También creo que es una estrategia de LaunchX para ver que tanta iniciativa propia tienen sus explorers. Animo, no pasa nada, solo seguir adelante :D
